CHANGELOG.next.asciidoc

```diff
@@ -86,6 +86,25 @@ https://github.com/elastic/beats/compare/v7.1.1...7.1[Check the HEAD diff]
 
 *Metricbeat*
 
+- Add AWS SQS metricset. {pull}10684[10684] {issue}10053[10053]
```

Comment:
These look unrelated. Were these entries included in this backport errantly?

Reply:
They were indeed 😞. Thanks for catching. I'll make a PR now to remove them.

Reply:
Here's the PR: https://github.com/elastic/beats/pull/12473.